test-suite/src/test/scala/org/scalajs/testsuite/javalib/StringTest.scala

```diff
@@ -98,6 +98,7 @@ object StringTest extends JasmineTest {
       expect("Scala.js".lastIndexOf("Scala.js")).toBe(0)
       expect("ananas".lastIndexOf("na")).toBe(3)
       expect("Scala.js".lastIndexOf("Java")).toBe(-1)
+      expect("Negative index".lastIndexOf("N", -5)).toBe(-1)
```

Comment:
I mean write a similar test for the overload of `lastIndexOf` that searches for an `Int` with a `fromIndex` (hence takes two `Int`s) -- as a separate `it`, below the one that says `it("should respond to `lastIndexOf(int)`") {`.